opm/simulators/wells/MultisegmentWell_impl.hpp

```diff
@@ -1802,6 +1805,9 @@ namespace Opm
                 }
                 perf_press_state[local_perf_index] = perf_press.value();
 
+                // mass rates, for now only water
+                perf_data.wat_mass_rates[local_perf_index] = cq_s[watCompIdx].value() * rhow;
```

Comment:
What happens here if water is not an active phase&ndash;e.g., in an oil/gas run?

Reply:
Now a check for water has been added, thanks again.